src/LessMsi.Gui/Windows.Forms/SearchPanel.cs

```diff
@@ -1,55 +1,60 @@
 ﻿using System;
-using System.Diagnostics;
 using System.Drawing;
 using System.Windows.Forms;
 
 namespace LessMsi.Gui.Windows.Forms
 {
 	internal partial class SearchPanel : UserControl
-	{
```

Comment:
Sorry, I had to rewrite quite some stuff here:

- The events would be registered every time a search dialog was opened, so I updated those to 'Action' instead so that they are replaced, not added.
- This was written to only bind to one grid in it's lifetime, I updated it to be able to re-register on another grid.